### ISSUE TITLE
Support for multiple rooms

### ISSUE THREAD

**Issue by salierri** [salierri/geltara#8]
If the app supported multiple groups, it could be used by a larger audience.
- Any user could be able to create their own room with only a name, and get an invite link (maybe make their room discoverable, but I don't really see the point of that, maybe for other use-cases than DnD).
- The rooms could exist for example 10 hours after the last user has left
- Rooms could have their own song collections (this feature would imply the existence of user accounts)
- An admin interface would be nice to track the CCU, the existing rooms, and the overall load on the server

In conclusion, this doesn't look like the near future...
